packages/gestalt/src/Link.js

```diff
@@ -30,13 +31,9 @@ type Props = {|
    */
   accessibilitySelected?: boolean,
   /**
-   * Link is a wrapper around components (or children), most commonly text, so that they become hyperlinks. See the [Link and Text](https://gestalt.pinterest.systems/link#Link-and-Text) variant for more information.
```

Comment:
You don't think this link is still useful?

Reply:
DONE